codespell_lib/data/dictionary.txt

```diff
@@ -5990,6 +6147,20 @@ authetnticators->authenticators
 authobiographic->autobiographic
 authobiography->autobiography
 authoer->author
+authoisation->authorisation
```

Comment:
Perhaps add `authorial` typos?

Reply:
`authoial->authorial` is too uncommon to be useful. The same goes for other `author` typos.